Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -40,59 +40,50 @@ public class PropertiesFileGeneratorTests
 
     [TestMethod]
     public void PropertiesFileGenerator_WhenConfigIsNull_Throws() =>
-        ((Action)(() => new PropertiesFileGenerator(null, new TestLogger()))).Should()
-            .ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("analysisConfig");
+        ((Func<PropertiesFileGenerator>)(() => new(null, new TestLogger()))).Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("analysisConfig");
```

Comment:
You can do this:
```suggestion
        FluentActions.Invoking(() => new PropertiesFileGenerator(null, new TestLogger())).Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("analysisConfig");
```
Or shorten it further with a static using.

Reply:
It's applied in the next PR to save CI run